src/script/common/c_converter.cpp

```diff
@@ -427,6 +427,19 @@ bool getintfield(lua_State *L, int table,
 }
 
 bool getintfield(lua_State *L, int table,
+		const char *fieldname, s8 &result)
+{
+	lua_getfield(L, table, fieldname);
+	bool got = false;
+	if(lua_isnumber(L, -1)){
```

Comment:
Space missing after `if` and before `{`, otherwise :+1:

Reply:
This code is copied from the function above with the last parameter type changed.  I'm still new so not sure which is preferred:

a) Follow the style of the surrounding code
b) Follow the style guide introducing inconstancy 
c) Reformat the rest of the file

Reply:
b)
Follow the code style guidelines but only for the lines of code you are working on or nearby lines.
It's a balance, a) would mean nothing gets corrected, and c) would make PRs too big and complex to review.
A lot of our code still has bad code style, we're slowly correcting it as we work on MT.